config/policy.go

```diff
@@ -607,6 +812,26 @@ func (p *Policy) MustRouteID() uint64 {
 	return id
 }
 
+func writeBool(hash *xxhash.Digest, b bool) {
+	if b {
+		hash.Write([]byte{1})
+	} else {
+		hash.Write([]byte{0})
+	}
+}
+
+func writeUint32(hash *xxhash.Digest, t uint32) {
+	var buf [4]byte
+	binary.LittleEndian.PutUint32(buf[:], t)
+	hash.Write(buf[:])
+}
+
+func writeInt32(hash *xxhash.Digest, t int32) {
+	var buf [4]byte
+	binary.LittleEndian.PutUint32(buf[:], *(*uint32)(unsafe.Pointer(&t)))
```

Comment:
I ran a benchmark and didn't see much difference between `*(*uint32)(unsafe.Pointer(&t))` and `uint32(t)`. Any other reason to do it this way?

Reply:
Oops, yeah this can just be uint32(t)

Reply:
This method is probably redundant anyway